src/sentry/integrations/slack/integration.py

```diff
@@ -103,32 +122,28 @@ def get_team_info(self, access_token):
         resp.raise_for_status()
         status_code = resp.status_code
         resp = resp.json()
+        # TODO: track_response_code won't hit if we have an error status code
         track_response_code(status_code, resp.get("ok"))
 
-        return resp["team"]
-
-    def get_identity(self, user_token):
```

Comment:
Do we not need this anymore?

Reply:
@EvanPurkhiser We don't. We get the user ID in the original response so we don't have to make another API call.